src/core/include/openvino/op/constant.hpp

```diff
@@ -12,7 +12,8 @@
 #    define WAS_OV_LIBRARY_DEFINED_CONSTANT
 #endif
 
-#include "ngraph/runtime/shared_buffer.hpp"
+#include "ngraph/util.hpp"
```

Comment:
Can be removed?

Reply:
I did it in https://github.com/openvinotoolkit/openvino/pull/22184, but the change requires a lot of fixes